ledger/ledger.go

```diff
@@ -463,11 +463,24 @@ func (l *Ledger) LookupWithoutRewards(rnd basics.Round, addr basics.Address) (ba
 	return data, validThrough, nil
 }
 
-// Totals returns the totals of all accounts at the end of round rnd.
-func (l *Ledger) Totals(rnd basics.Round) (ledgercore.AccountTotals, error) {
+// LatestTotals returns the totals of all accounts for the most recent round, as well as the round number.
+func (l *Ledger) LatestTotals() (basics.Round, ledgercore.AccountTotals, error) {
 	l.trackerMu.RLock()
 	defer l.trackerMu.RUnlock()
-	return l.accts.Totals(rnd)
+	rnd := l.blockQ.latest()
+	totals, err := l.accts.Totals(rnd)
```

Comment:
why aren't we using `LatestTotals()`?

Reply:
the l.accts doesn't have LatestTotals().

Reply:
Right, you added `latestTotalsImpl()` to `accountUpdates`. We can just make it public?

Reply:
no, there is a need for a `au.LatestTotals`, which would call `au.latestTotalsImpl` while taking the read mutex.